vulcan/lib/server/workflows/scripts/parse_record_selections.py

```diff
@@ -6,8 +6,7 @@
 pdat = input_json("project_data")[project_name]
 selection_atts = pdat['selection_options']
 
-# because a is the input name from the previous CWL input step
-selected = input_json('selected_options')["a"]
+selected = input_json('selected_options')
```

Comment:
@dtm2451 @coleshaw @graft This is the change in the workflow to support the flattened data approach.